annotation/management/commands/download_annotations.py

```diff
@@ -30,9 +30,11 @@ def handle(self, *args, **options):
             # only download annotations for sound that are not the reference of the exercise
             if sound != sound.exercise.reference_sound:
                 try:
-                    # take the original filename as the destination in which to download the file
-                    annotation_file_path = os.path.splitext(sound.original_filename.replace('import', 'export'))[0]\
-                                           + '.json'
+                    # remove two first directories from original_filename name and add /export
+                    export_path_list = ['/export'] + sound.original_filename.split(os.sep)[2:]
```

Comment:
it'd be nice to put this in settings.
you can use os.path.join to join the directory parts together. Instead of using split to turn it into a list, you could use original_filename[:original_filename.find(os.sep, 1)]

Reply:
hey, the command original_filename[:original_filename.find(os.sep, 1)] just gives the first directory, so '/import' in most cases